src/apiCalls.js

```diff
@@ -1,9 +1,9 @@
 // Your fetch requests will live here!
 const fetchApiUrl = (path) => {
   return fetch(`https://fitlit-api.herokuapp.com/api/v1/${path}`)
```

Comment:
You could look into `async` and `await` here. This works fine, but that's a more modern way to write this kind of thing.

Reply:
We have been told not to use async/await